tools/cibadmin.c

```diff
@@ -732,7 +732,16 @@ main(int argc, char **argv)
         goto done;
     }
 
-    if (strcmp(options.cib_action, "md5-sum") == 0) {
+    if (options.cib_action == cib_action_EMPTYCIB  /* pointer comparison */) {
+        output = createEmptyCib(1);
+        if (optind < argc) {
+            crm_xml_add(output, XML_ATTR_VALIDATION, argv[optind]);
+        }
+        pcmk__xml_serialize_fd_formatted(STDOUT_FILENO, output);
+        xmlFreeDoc(output->doc);
+        crm_exit(CRM_EX_OK);
+
+    } else if (pcmk__str_eq(options.cib_action, "md5-sum", pcmk__str_casei)) {
```

Comment:
This is now handled by the `strcmp(options.cib_action, "empty")` block instead

crm_diff.c:print_patch(), crm_ticket.c:dump_ticket_xml(), and crm_ticket.c:dump_constraints() should be able to use the new function, too.